server/datastore/mysql/labels_test.go

```diff
@@ -673,6 +673,33 @@ func testLabelsChangeDetails(t *testing.T, db *Datastore) {
 	saved, _, err := db.Label(context.Background(), label.ID, filter)
 	require.Nil(t, err)
 	assert.Equal(t, label.Name, saved.Name)
+	assert.Equal(t, label.Description, saved.Description)
+
+	// Create an Apple config profile, which should reflect a change in label's name
+	profA, err := db.NewMDMAppleConfigProfile(context.Background(), *generateCP("a", "a", 0))
+	require.NoError(t, err)
+	ExecAdhocSQL(t, db, func(q sqlx.ExtContext) error {
+		_, err := q.ExecContext(context.Background(),
+			"INSERT INTO mdm_configuration_profile_labels (apple_profile_uuid, label_name, label_id) VALUES (?, ?, ?)",
+			profA.ProfileUUID, label.Name, label.ID)
+		return err
+	})
+	label.Description = "changed name"
```

Comment:
`label.Name`?

Reply:
@lucasmrod Updated, and fixed another bug. Please re-review.